src/containers/tutor-profile/profile-info/ProfileContainerDesktop.jsx

```diff
@@ -1,46 +1,58 @@
 import Box from '@mui/material/Box'
 import SchoolIcon from '@mui/icons-material/School'
 import DoneIcon from '@mui/icons-material/Done'
+import Avatar from '@mui/material/Avatar'
+
+import _ from 'lodash'
 
 import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
 import AppChipList from '~/components/app-chips-list/AppChipList'
 import ProfileDoneItemsList from '~/components/icon-with-text-list/ProfileDoneItemsList'
 import useBreakpoints from '~/hooks/use-breakpoints'
 
-import img from '~/assets/img/tutor-profile-page/avatar.png'
 import { styles } from '~/containers/tutor-profile/profile-info/ProfileInfo.styles'
 
 const ProfileContainerDesktop = ({
+  userData,
   actionIcon,
   accInfo,
   buttonGroup,
   defaultQuantity,
-  subjectChips,
   doneItems
 }) => {
   const { isDesktop } = useBreakpoints()
 
+  const subjectData = userData.mainSubjects.tutor.map((item) => item.name)
+
+  const profilePhoto = _.isEmpty(userData.photo) ? (
```

Comment:
what are empty stands for? photo is just a string, use just Avatar. not ternary

Reply:
I cannot use Avatar all the time. If user has a photo then Avatar will contain a photo of the user and will have a round shape. As a result, I will have to change the styles of Avatar component. I don't see it reasonable.

Reply:
have u tried to give props to it? maybe some styles

Reply:
I have changed only the sizes of the Avatar. Let's me review docs again

Reply:
Done.